ASM/c/debug.c

```diff
@@ -31,71 +38,217 @@ warp_t dungeon_warps[] = {
     { 12, 0x467, "Ganon Castle"},
 };
 
+room_t deku_rooms[6] = {
+    {  0x0, 0, { -4, 0, 554 }, 32768, "Entrance"},
+    {  0x0, 2, { -1002, 400, 1000 }, 57436, "Slingshot room"},
+    {  0x0, 10, { -651, 800, -1 }, 49050, "Compass room"},
+    {  0x0, 3, { -198, -845, -285 }, 48980, "Basement"},
+    {  0x0, 5, { -428, -880, 961 }, 49050, "Log Spike"},
+    {  0x0, 9, { -614, -1881, -388 }, 34300, "231 Scrubs"},
+};
+
+room_t dc_rooms[] = {
+    {  0x04, 0, { -3, 0, 369 }, 32768, "Entrance"},
+    {  0x04, 2, { -927, 0, -1525 }, 49050, "Bomb Flower Staircase"},
+    {  0x04, 9, { 1392, 531, -316 }, 16384, "Bomb Bag room"},
+    {  0x04, 7, { 7, 76, -2057 }, 32768, "Floor switch room before Boss"},
```

Comment:
Not a huge deal but I'm gonna note what room names overflow the textbox. Wrapping them would be cool but I imagine that's not nearly as easy as it sounds lol

Reply:
So i've took the lazy way and just increased the width of the window when you're in the rooms section.